content/src/controller/Controller.ts

```diff
@@ -93,9 +92,7 @@ export class Controller {
     } else {
       entities = await this.components.deployer.getEntitiesByPointers(type, pointers)
     }
-    const maskedEntities: ControllerEntity[] = entities.map((entity) =>
-      ControllerEntityFactory.maskEntity(entity, enumFields)
-    )
+    const maskedEntities: Entity[] = entities.map((entity) => ControllerEntityFactory.maskEntity(entity, enumFields))
```

Comment:
I could validate it doesn't modifies the response of `/entities/:type` since we are using same type but `content` mapping is not required anymore.

It filters the fields declared in `enumFields` from the response entity, so I think it's ok to leave it.